app/src/main/java/com/cmput301f18t20/medicalphotorecord/Problem.java

```diff
@@ -3,15 +3,15 @@
 import android.location.Location;
 
 import java.util.ArrayList;
+import java.util.Date;
 
 public class Problem extends Entry implements Refreshable {
     protected String description;
     protected ArrayList<Record> records = new ArrayList<>();
 
-    Problem() {
+    public Problem(String createdByUserID, String title) {
         super();
```

Comment:
Needs to call super(createdByUserID, title);

Reply:
Should be fixed